.github/workflows/buildTest.yaml

```
@@ -7,11 +7,8 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
   push:
-    branches:
-    - master
-  pull_request:
-    branches:
-    - master
+    tags: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)